### DISCUSSION TITLE
Disable on change motion or visually selecting

### DISCUSSION THREAD

**Discussion by rafaelrinaldi** [ggandor/leap.nvim#249] [Q&A] [answered]
Is it possible to only get Leap to trigger when navigating code via `s` or `S`? Sometimes I am just running a "change inside word" or a "visualy select around brackets" and I need to hit an extra key stroke because Leap gets triggered and needs me to chose something to continue.

**Comment by ggandor** [comment, marked as answer]
> Sometimes I am just running a "change inside word" or a "visualy select around brackets" and I need to hit an extra key stroke because Leap gets triggered

That should not happen, you should check your config/mappings. You might have accidentally remapped your native text objects with "remote" text objects (https://github.com/ggandor/leap.nvim?tab=readme-ov-file#extras) (?)
